src/js/components/projects/createProjectModal/form.tsx

```diff
@@ -100,7 +100,8 @@ const CreateProjectForm = ({
             slds-grow
             slds-shrink-none
             slds-grid_align-end
-            slds-grid_vertical-align-center"
+            slds-m-top_large
+            slds-grid_vertical-align-start"
```

Comment:
This gets the "Re-Sync GitHub Organizations" button to line up better with the GitHub Organization dropdown. 

Before: 
![bild](https://user-images.githubusercontent.com/41588129/170102777-4e4f7546-4159-4b05-a938-1a116f27761e.png)

![bild](https://user-images.githubusercontent.com/41588129/170102893-7e6a92e8-2567-4f5b-8dd5-5cf6e7c761e5.png)

After:
![bild](https://user-images.githubusercontent.com/41588129/170102955-4d2996e9-b830-4048-874e-a305eab27662.png)

![bild](https://user-images.githubusercontent.com/41588129/170103024-a5866168-a142-4c8b-a21c-0c7b04fa9347.png)

The "You are not a member. . ." text gets a little long on really narrow screens. We can adjust that if needed.